metrics/metrics.go

```diff
@@ -0,0 +1,32 @@
+package metrics
+
+import metrics "github.com/rcrowley/go-metrics"
+
+type Metrics struct {
+	registry metrics.Registry
+}
+
+const (
+	clientCounterName = "counter"
+)
+
+func New() (Metrics, error) {
+	m := Metrics{
+		registry: metrics.NewRegistry(),
+	}
+
+	clientCounter := metrics.NewCounter()
+	if err := m.registry.Register(clientCounterName, clientCounter); err != nil {
+		return m, err
+	}
+
+	return m, nil
+}
+
+func (m *Metrics) IncClientCount() {
```

Comment:
__[golint]__
* [golint] exported method Metrics.IncClientCount should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->